modules/tfm/tfm/boards/common/assert.c

```diff
@@ -11,10 +11,18 @@
 
 #define IS_ALIGNED_POW2(value, align) (((value) & ((align) - 1)) == 0)
 
-/* Making sure the borders between secure and non-secure regions are aligned with the SPU regions */
+#define ADDRESS_TO_STR_HELPER(addr) #addr
+#define ADDRESS_TO_STR(addr)        ADDRESS_TO_STR_HELPER(addr)
```

Comment:
There is an M2S macro in utilities.h (TF-M header).
We could also define one as part of our Zephyr definition support headers, in toolchain/common.h it is defined as STRINGIFY.

Reply:
It's already there apparently! 
I will update to use STRINGIFY